letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -264,7 +264,18 @@ UNLIKELY_EOF
       # Report error. (Otherwise, be quiet.)
       echo "Had a problem while installing Python packages."
       if [ "$VERBOSE" != 1 ]; then
+        echo
+        echo "pip prints the following errors: "
+        echo "====================================================="
         echo "$PIP_OUT"
+        echo "====================================================="
+        echo
+        echo "Certbot has problem setting up the virtual environment."
```

Comment:
I've just recreated the problem on my DO droplet again. On Ubuntu it does not seems to print `allocate memory`, just `Killed`.

@pde Should we grep for `Killed` ***or*** `allocate memory`?